ui-v2/app/controllers/dc/intentions/edit.js

```diff
@@ -23,7 +25,7 @@ export default Controller.extend({
     this._super({
       ...model,
       ...{
-        item: this.changeset,
+        item: this.form.setData(model.item).getData(),
         SourceName: source,
         DestinationName: destination,
       },
```

Comment:
This one does use `Object.assign` so I need to take a second look and make them all as similar as possible